LibVLCSharp.Uno/VideoView.iOS.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace LibVLCSharp.Platforms.UWP
```

Comment:
Don't we want a `LibVLCSharp.Uno.iOS` namespace here?

Reply:
Not that's possible...

Reply:
No, the namespaces should exactly the same between the UWP version and Android or iOS. With Uno, you take your UWP code and it must work without modification in your Android and iOS app.
In fact, whe should rename the namespaces in LibVLCSharp to remove the `Platforms.UWP` but it will be a breaking change.

Reply:
Still. I don't get why you need to use the existing `LibVLCSharp.Platforms.UWP` namespace. What prevents you from putting all the Uno code in `LibVLCSharp.Uno`?

Reply:
Because there is no "Uno" version of the VideoView for UWP, we reuse the LibVLCSharp.Platforms.UWP.VideoView.

Reply:
No, you can't do this. Otherwise, the LibVLCSharp.Uno.Sample.UWP project will not compile.

Reply:
Indeed, but it should still be possible. Also why not provide a UAP build? reactiveui seems to have one (along with a custom namespace) https://github.com/reactiveui/ReactiveUI/blob/master/src/ReactiveUI.Uno/ReactiveUI.Uno.csproj

Reply:
Because in our case, I don't need anything from `LibVLCSharp.Uno` when I create an UWP application.
An UWP developer will not include `LibVLCSharp.Uno` because he doesn't need it, `LibVLCSharp` is enough.
It's only when he wants to port its application on Android or iOS that he will use `LibVLCSharp.Uno`. No work to do, he takes the UWP code and it must work without modifying the namespaces.

Reply:
Still... :)
Re-using `LibVLCSharp.UWP` directly wrecks the namespace for LibVLCSharp with Uno on Android and iOS. This will be very confusing for users and I don't see the benefit.
We should provide a UAP build, even if it is only a thin shell over `LibVLCSharp.UWP`, and that will allow proper namespace usage.

Reply:
Still too... :)
Yes, I can do this but I don't think it's a good idea. Even there is no benefit, it's not the "Uno Platform spirit" :)
In fact, Uno doesn't exist for UWP and you should not have any Uno library in your UWP project. You have your existing current code for UWP, and it should remain the same when you want to port your app in Android or iOS.
IMO, the real change that should be done is to remove this .Platform.XXX in the namespace in the LibVLCSharp library because it's useless. But, this would introduce a breaking change.

Reply:
Finally, I was thinking about where I will place the MediaPlayerElement. As the code will be exactly the same for all platforms, I will not doing one library for UWP and another for Uno Android and iOS. The best place for the code of the MediaPlayerElement will be the LibVLCSharp.Uno project. So, in this case, we will be obliged to add a reference of the LibVLCSharp.Uno in our UWP project. I'm going to make an exception by adding an UWP target in this Uno library and take the opportunity to change the namespace (for UWP, I will just inherit of the VideoView and change the namespace).

Reply:
I did it in e218b978878e1635391cb8297d2375210ba0ee6a.